tests/language_packs/test_language_pack_install_addons.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from selenium.webdriver import Firefox
+
+from modules.page_object import AboutAddons, AboutPrefs, GenericPage
+
+LANGUAGE_ADDONS_LINK = "https://addons.mozilla.org/en-US/firefox/language-tools/"
+LANGUAGES = [("Italiano", "it", "Imposta alternative…")]
+
+
+@pytest.mark.parametrize("drop_down_name, shortform, localized_text", LANGUAGES)
+def test_language_pack_install_from_addons(
+    driver: Firefox, drop_down_name: str, shortform: str, localized_text: str
+):
+    """
+    C1549408: verify that installing a language pack from about:addons will correctly change the locale
+    """
+    # declaring objects and navigating
+    generic_page = GenericPage(driver)
+    driver.get(LANGUAGE_ADDONS_LINK)
+
+    # ensuring the page was loaded
+    generic_page.custom_wait(timeout=20).until(
+        lambda _: generic_page.get_element("language-addons-title") is not None
+    )
+
+    # grab the appropriate link and wait until the page is loaded
+    language_row = generic_page.get_element("language-addons-row")
```

Comment:
For many of these steps, they could be abstracted into methods on a new POM, but how many and which is a question I'll leave to you...